test/20-multiple-config.js

```diff
@@ -56,4 +56,17 @@ vows.describe('Tests for multiple config')
 
   }
 })
+.addBatch({
+  'Empty string should not blow up': function () {
+    process.env.NODE_CONFIG_DIR =  [
+      './test/20-config',
+      ''
```

Comment:
I don't usually like doing this, but sometimes you end up with batch files that end up evaluating a conditional entry to "" and then you're screwed if you're tucking it into NODE_CONFIG_DIR, so I'd rather not have the cryptic error message even if I'd prefel people scrub their inputs. At least the new code should make it a bit more obvious in a step-wise debug of 'where did my configs go?'

Reply:
OK, we can refine later if people complain about the lack of warning message.